chapter_12_organising_test_files.asciidoc

```diff
@@ -874,6 +855,7 @@ Use a tests folder::
       'test_forms.py'.
 // DAVID: worth mentioning that mirroring test files to real files can pave the way for automatically running tests
 // relevant to your changes.
+// CSANAD: TODO ask David to elaborate
```

Comment:
@seddonym Could you please explain what you mean by this specifically? Or maybe would you like to try phrasing it yourself?

Reply:
We have git hooks that automatically run test files based on the changed files. So for example if you change a file called `path/to/somemodule.py` and the tests are located in (say) `tests/path/to/test_somemodule.py` then we can automatically run those tests.

@hjwp will be familiar with this as it's something we do at our workplace.

Reply:
indeed!  i'm not sure i'm really a fan of the whole idea tho.